kubernetes/yamls/web/ingress-tls.yml

```diff
@@ -0,0 +1,21 @@
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: web
+  namespace: web
+  annotations:
+    kubernetes.io/tls-acme: "true"
+    kubernetes.io/ingress.class: "nginx"
+spec:
+  tls:
+  - hosts:
+    - api.loklak.org
```

Comment:
Also, start with staging (dev) deployment first. If things go well, we'll add this to api (master) deployment.